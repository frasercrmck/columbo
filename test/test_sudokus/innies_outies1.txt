# RUN: columbo -q -s innies-outies -f %s
# RUN: columbo -q -s innies-outies -f %s -o - | columbo_check %S/expected_outputs/innies_outies1.txt
# RUN: columbo -q -s innies-outies -s innies-outies -f %s -o - | columbo_check %S/expected_outputs/innies_outies1x2.txt
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF
0x1B7 0x1B7 0x1B7 0x1F7 0x1F7 0x1F7 0x008 0x1F7 0x1F7
0x1BF 0x1BF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1F7 0x1F7
0x1BF 0x1BF 0x040 0x1BF 0x1BF 0x1BF 0x1B7 0x1B7 0x1B7
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF
0x1FF 0x1FF 0x1BF 0x1FF 0x1FF 0x1FF 0x1F7 0x1FF 0x1FF

9  A0 A1
7  A2 A3
18 A4 A5 B5
22 A6 A7 B6
14 A8 B7 B8
12 B0 C0 D0
9  B1 C1
14 B2 B3 B4
15 C2 D2 E2
19 C3 C4 D3
4  C5 C6
6  C7 C8
15 D1 E0 E1
20 D4 D5 E5
4  D6
13 D7 D8
16 E3 E4 F3 F4
15 E6 F6 G6
17 E7 E8 F7
14 F0 F1
7  F2
15 F5 G4 G5
10 F8 G8 H8
6  G0 G1
12 G2 G3
17 G7 H7
16 H0 H1 J0
18 H2 J1 J2
12 H3 J3 J4
11 H4 H5 H6
11 J5 J6
7  J7 J8
