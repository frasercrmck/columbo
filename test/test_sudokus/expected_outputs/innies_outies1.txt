0x1df 0x0df 0x00d 0x04f 0x1df 0x1d7 0x1d3 0x020 0x1db
0x01b 0x01b 0x0ac 0x1ef 0x1ff 0x1f7 0x1d3 0x1db 0x1db
0x1fb 0x0f8 0x180 0x1eb 0x1fb 0x1f3 0x004 0x1db 0x1db
0x006 0x0a7 0x1a7 0x010 0x1e7 0x1e7 0x008 0x1c7 0x1e7
0x1bf 0x0bf 0x1af 0x1e7 0x1e7 0x1e7 0x1f3 0x1d7 0x1f7
0x1b7 0x0b7 0x040 0x1a7 0x1a7 0x008 0x1b3 0x197 0x017
0x0ef 0x0ef 0x010 0x1ef 0x1ef 0x1e7 0x160 0x1cc 0x1ef
0x0ef 0x0ef 0x0af 0x1ef 0x1ff 0x1f7 0x013 0x180 0x060
0x0ef 0x100 0x0af 0x0ef 0x0ff 0x070 0x032 0x0df 0x0ff

9  A0 A1
7  A2 A3
18 A4 A5 B5
22 A6 A7 B6
14 A8 B7 B8
12 B0 C0 D0
9  B1 C1
14 B2 B3 B4
15 C2 D2 E2
19 C3 C4 D3
4  C5 C6
6  C7 C8
15 D1 E0 E1
20 D4 D5 E5
4  D6
13 D7 D8
16 E3 E4 F3 F4
15 E6 F6 G6
17 E7 E8 F7
14 F0 F1
7  F2
15 F5 G4 G5
10 F8 G8 H8
6  G0 G1
12 G2 G3
17 G7 H7
16 H0 H1 J0
18 H2 J1 J2
12 H3 J3 J4
11 H4 H5 H6
11 J5 J6
7  J7 J8
